system/Common.php

```diff
@@ -163,8 +163,31 @@ function command(string $command)
 	{
 		$runner = service('commands');
 
-		$params  = explode(' ', $command);
-		$command = array_shift($params);
+		$args    = explode(' ', $command);
+		$command = array_shift($args);
+
+		$params = [];
+		for ($i = 0, $c = count($args); $i < $c; $i++)
```

Comment:
can use foreach which is faster, ref https://phpsandbox.io/n/super-fog-72g7 

```bash
For took: 0.069ms
For with count() took: 0.442ms
Foreach took: 0.132ms
```

Reply:
Ok. But how will I check the next item `[$i + 1]` using `foreach`? I tried foreach first but stumbled here.

Reply:
well, you can do: 

```php
foreach ($args as $i => $arg)
{
    if ($args[$i+1] === 'foo') 
   {

   }
}
```

Reply:
Alright. I forgot that `foreach` has another syntax. Will change.